github_strategy.md

```diff
@@ -0,0 +1,166 @@
+
+We use software libraries in these analyses, and we save specific versions in conda environment files or docker containers. We also must specify the license, present a clear README with reproducibility/usage instructions, a brief summary of the results, and citation instructions.
+
+Analysis repositories must be included in our publications, and we consider them to be the ground truth methods section. The analyses must be fully reproducible (we aim for [Gold Reproducibility](https://www.nature.com/articles/s41592-021-01256-7#Sec4)) and open source. For each submission and subsequent resubmission, we generate specific “github releases”. We generate direct object identifiers (DOI) for these releases using Zenodo ([instructions](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content)). We do not restrict ourselves to the one-repository per-paper strategy, instead, we opt for the one-release per-submission strategy (which can span multiple publications). 
+
+Examples: [phenotypic\_profiling\_model](https://github.com/WayScience/phenotypic_profiling_model), [gene\_dependency\_representations](https://github.com/WayScience/gene_dependency_representations)
+
+### 3. Data repository
+
+In many projects, we collaborate with labs that generate data directly, and it is up to us to apply bioinformatics pipelines to generate analysis-ready data. We also often re-analyze existing, publicly available data (e.g. the resources on [Image Data Resource](https://idr.openmicroscopy.org/)). In these cases, we use a third github repository design: A data repository.
+
+The data repository contains two key elements: 1) A bioinformatics pipeline, or recipe, of all computational steps applied to the data and 2) the data, or pointers to the data. We strive to release all of our bioinformatics steps and data publicly so that others (and ourselves!) can benefit. We believe that version controlling both our pipeline and data is essential for reproducibility, and so that we (and others) understand exactly what steps generated all intermediate files. We also make sure that this link between the pipeline and data is itself version-controlled, so that data processing steps do not lose sync in re-analysis.
+
+This process ensures that our data can be easily reprocessed by updated bioinformatics methods with new and improved methods.
+
+Examples: [cell-health-data](https://github.com/WayScience/cell-health-data), [nf1\_cellpainting\_data](https://github.com/WayScience/nf1_cellpainting_data), [CFReT\_data](https://github.com/WayScience/CFReT_data)
```

Comment:
Might consider a note for CFReT that this repo is a bit non-traditional to the WayLab standard since both analysis and data are held here. Might be useful to show the differences and the pros and cons to both methods. Completely optional.

Reply:
I'm considering changing both of these to more recent examples (see above).